ray-operator/test/support/yaml.go

```diff
@@ -52,6 +52,8 @@ func KubectlApplyYAML(t Test, filename string, namespace string) {
 	t.T().Helper()
 	kubectlCmd := exec.CommandContext(t.Ctx(), "kubectl", "apply", "-f", filename, "-n", namespace)
 	err := kubectlCmd.Run()
-	assert.NoError(t.T(), err)
-	t.T().Logf("Successfully applied %s", filename)
+	if err != nil {
+		t.T().Fatalf("Failed to apply %s to namespace %s: %v", filename, namespace, err)
+	}
```

Comment:
Shouldn't it be `assert.NoError(t.T(), err, errMsg)`?

Reply:
If we use `assert.NoError(t.T(), err, errMsg)`, the test will continue executing and report the error only after the test finishes. `t.T().Fatalf(...)` will fail immediately.